supabase/migrations/15_oauth.sql

```diff
@@ -0,0 +1,28 @@
+-- OAuth Client ID and Client Secret are associated uniquely with
```

Comment:
Check out [this chunk of the readme](https://github.com/estuary/animated-carnival#production-migrations) -- in essence, rather than adding a new migration here, you should be updating the existing `connectors` "create table" statement.

Then, add an actual migration under `supabase/pending` which will migrate the production DB from the current schema to your updated one. This migration is checked in and PR'd but is ephemeral and will be removed once applied to the DB, leaving only the pristine schema.

Check out [this PR](https://github.com/estuary/animated-carnival/pull/31/files#diff-b55bd531e066504f1610cc8f00e42cda4972a5751ae99fc1388c7cf38d0f5da9) for an example of this workflow.